From 6834f1ac5c2228f1439e4a0e72f56623eb56dd67 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Tue, 9 May 2023 15:32:05 +0530
Subject: [PATCH] Keep Default AAOS Behavior with Multi-Display Launcher.

Enabling multi-display Secondary Home Launcher
for secondary displays.

Tracked-On: OAM-109879
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 core/res/res/values/config.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index f9b951e92736..c393aaded175 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -4219,11 +4219,11 @@
          This launcher package must have an activity that supports multiple instances and has
          corresponding launch mode set in AndroidManifest.
          {@see android.view.Display#FLAG_SHOULD_SHOW_SYSTEM_DECORATIONS} -->
-    <string name="config_secondaryHomePackage" translatable="false">com.android.launcher3</string>
+    <string name="config_secondaryHomePackage" translatable="false">com.android.car.multidisplay</string>
 
     <!-- Force secondary home launcher specified in config_secondaryHomePackage always. If this is
          not set, secondary home launcher can be replaced by user. -->
-    <bool name ="config_useSystemProvidedLauncherForSecondary">false</bool>
+    <bool name ="config_useSystemProvidedLauncherForSecondary">true</bool>
 
     <!-- If device supports corner radius on windows.
          This should be turned off on low-end devices to improve animation performance. -->
-- 
2.17.1

